src/index.js

```diff
@@ -151,33 +151,38 @@ class Plugin {
 	getLogMetricCF(alarm, functionName, normalizedFunctionName){
 		var output = {};
 		if (alarm.pattern) {
-			//add custom log metric
-			const logMetricCFRef = this.naming.getLogMetricCFRef(normalizedFunctionName,alarm.name)+'ALERT';
+			const logMetricCFRefBase = this.naming.getLogMetricCFRef(normalizedFunctionName,alarm.name);
+			const logMetricCFRefALERT = `${logMetricCFRefBase}ALERT`;
+			const logMetricCFRefOK = `${logMetricCFRefBase}OK`;
 			const CFLogName = this.serverless.getProvider('aws').naming.getLogGroupLogicalId(functionName);
-			output[logMetricCFRef] = {
+			const metricNamespace = `${this.serverless.service.service}_${alarm.namespace}`;
+			const logGroupName =  this.serverless.getProvider('aws').naming.getLogGroupName(this.serverless.service.getFunction(functionName).name);
+
+			//add custom log metric for alert state
+			output[logMetricCFRefALERT] = {
 				Type: "AWS::Logs::MetricFilter",
 				DependsOn: CFLogName,
 				Properties: {
 					FilterPattern: alarm.pattern,
-					LogGroupName: this.serverless.getProvider('aws').naming.getLogGroupName(this.serverless.service.getFunction(functionName).name),
+					LogGroupName: logGroupName,
 					MetricTransformations: [{
 						MetricValue: 1,
-						MetricNamespace: `${this.serverless.service.service}_${alarm.namespace}`,
+						MetricNamespace: metricNamespace,
 						MetricName: alarm.metric + normalizedFunctionName
```

Comment:
it should be. I need them both to post the the same metric namespace and metric name. one posting 0 and the other 1.